<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="FrameContenedor">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="ContenedorInfoBloque" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>851</width>
        <height>411</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="nBloque">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>231</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bloqueEjecucion">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>571</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="TablaBloques">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>811</width>
         <height>241</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Order</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Resultado</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Operador</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QFrame" name="ContenedorInfoPaso">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>409</y>
        <width>851</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="nPasos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 28pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="descripcionPaso">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>471</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="resultadoPaso">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>10</y>
         <width>231</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>811</width>
         <height>341</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>809</width>
          <height>339</height>
         </rect>
        </property>
        <widget class="QTableWidget" name="TablaPasos">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>811</width>
           <height>321</height>
          </rect>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="autoScrollMargin">
          <number>1</number>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashDotDotLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>9</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>129</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>40</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Orden</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Estado</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rango Minimo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rango Maximo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Resultado</string>
          </property>
         </column>
         <item row="2" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="ContenedorAccion" native="true">
      <property name="geometry">
       <rect>
        <x>849</x>
        <y>-1</y>
        <width>461</width>
        <height>841</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>381</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QRadioButton" name="Automatico">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Automatico</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="Automatico_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Manual</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ContinuarLuego">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Continuar luego</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abortar">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Abortar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bloqueNo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Bloque NO Corresponde</string>
        </property>
       </widget>
       <widget class="QPushButton" name="itemNo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Item NO Corresponde</string>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="numSerie">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="TablaEstadistica">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>420</y>
         <width>311</width>
         <height>71</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>#PASOS</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>EJECUTADOS</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ERRORES</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
